app/assets/scss/views/_service_updates.scss

```diff
@@ -0,0 +1,19 @@
+.govuk-option-select {
```

Comment:
That's a good point. I took these styles from the buyers app so they are shared so suitable for pushing to the toolkit.

I'll raise a pull request to that end (which will involve moving the buyers app to a toolkit version too) though I don't see why it should hold up this one.